Flaky PGJobContextTest.testCancelRunningQuery

### Describe the bug

https://dev.azure.com/questdb/questdb/_build/results?buildId=19351&view=logs&j=2a8db746-6336-53bd-306b-3f5b3836e6eb&t=856b52a7-8d37-5a6e-f64c-b8e55c77b9fc

[testfail.log](https://github.com/questdb/questdb/files/11211097/testfail.log)


### To reproduce

_No response_

### Expected Behavior

_No response_

### Environment

```markdown
- **QuestDB version**:
- **OS**:
- **Browser**:
```


### Additional context

_No response_

Occurred again on windows-other:
```
2023-11-16T13:21:27.6294456Z 2023-11-16T13:21:27.628446Z I i.q.t.TestListener >>>> io.questdb.test.cutlass.pgwire.PGJobContextTest.testCancelRunningQuery[NO_WAL]
2023-11-16T13:21:27.6296203Z 2023-11-16T13:21:27.628497Z I i.q.t.c.p.PGJobContextTest random seeds: 1737332879100L, 1700140887614L
2023-11-16T13:21:27.6299835Z 2023-11-16T13:21:27.629305Z I i.q.t.AbstractTest Starting test PGJobContextTest#testCancelRunningQuery[NO_WAL]
2023-11-16T13:21:27.6308169Z 2023-11-16T13:21:27.630507Z I i.q.t.s.TestFilesFacadeImpl cannot remove file: C:\Users\VSSADM~1\AppData\Local\Temp\junit16836771537375288158\dbRoot\tables.d.0, errno:2
2023-11-16T13:21:27.6320147Z 2023-11-16T13:21:27.631720Z I i.q.c.TableNameRegistryStore reloading tables file [path=C:\Users\VSSADM~1\AppData\Local\Temp\junit16836771537375288158\dbRoot\tables.d.0]
2023-11-16T13:21:27.6325517Z 2023-11-16T13:21:27.631900Z I i.q.t.c.p.PGJobContextTest fragmentation params [sendBufferSize=512, forceSendFragmentationChunkSize=15, recvBufferSize=5120, forceRecvFragmentationChunkSize=411]
2023-11-16T13:21:27.6328487Z 2023-11-16T13:21:27.631965Z I i.q.t.c.p.PGJobContextTest asserting PG Wire server [mode=EXTENDED, binary=true, prepareThreshold=-2]
2023-11-16T13:21:27.6330703Z 2023-11-16T13:21:27.632005Z I i.q.t.AbstractTest Starting test PGJobContextTest#testCancelRunningQuery[NO_WAL]
2023-11-16T13:21:27.6348953Z 2023-11-16T13:21:27.633905Z I i.q.c.TableNameRegistryStore reloading tables file [path=C:\Users\VSSADM~1\AppData\Local\Temp\junit16836771537375288158\dbRoot\tables.d.0]
2023-11-16T13:21:27.6352345Z 2023-11-16T13:21:27.634318Z I i.q.g.SqlCompilerImpl parse [fd=-1, q=CREATE TABLE IF NOT EXISTS "sys.text_import_log" (ts timestamp, id string, table_name symbol, file symbol, phase symbol, status symbol, message string,rows_handled long,rows_imported long,errors long) timestamp(ts) partition by DAY BYPASS WAL]
2023-11-16T13:21:27.6357459Z 2023-11-16T13:21:27.635033Z I i.q.c.CairoEngine locked [table=`sys.text_import_log~`, thread=8548]
2023-11-16T13:21:27.6669870Z 2023-11-16T13:21:27.665313Z I i.q.c.p.WriterPool created [table=`sys.text_import_log~`, thread=8548]
2023-11-16T13:21:27.6672387Z 2023-11-16T13:21:27.665345Z I i.q.c.TableWriter open 'sys.text_import_log'
2023-11-16T13:21:27.6785622Z 2023-11-16T13:21:27.673926Z I i.q.c.CairoEngine unlocked [table=`sys.text_import_log`]
2023-11-16T13:21:27.6787196Z 2023-11-16T13:21:27.673961Z I i.q.c.p.WriterPool >> [table=`sys.text_import_log~`, thread=8548]
2023-11-16T13:21:27.6788341Z 2023-11-16T13:21:27.674202Z A pg-server listening on 0.0.0.0:0 [fd=17884 backlog=64]
2023-11-16T13:21:27.6789445Z 2023-11-16T13:21:27.674443Z I i.q.t.c.p.PGJobContextTest worker pool started [pool=testing]
2023-11-16T13:21:27.6790627Z 2023-11-16T13:21:27.675277Z I i.q.t.c.p.PGJobContextTest os scheduled worker started [name=testing_0]
2023-11-16T13:21:27.6791860Z 2023-11-16T13:21:27.675303Z I i.q.t.c.p.PGJobContextTest os scheduled worker started [name=testing_1]
2023-11-16T13:21:27.6792934Z 2023-11-16T13:21:27.675512Z I pg-server connected [ip=127.0.0.1, fd=5564]
2023-11-16T13:21:27.6794030Z 2023-11-16T13:21:27.676009Z I i.q.g.SqlCompilerImpl parse [fd=5564, q=SET extra_float_digits = 3]
2023-11-16T13:21:27.6795815Z 2023-11-16T13:21:27.676130Z I i.q.g.SqlCompilerImpl parse [fd=5564, q=SET application_name = 'PostgreSQL JDBC Driver']
2023-11-16T13:21:27.6797991Z 2023-11-16T13:21:27.676299Z I i.q.g.SqlCompilerImpl parse [fd=-1, q=create table if not exists tab as (select x::timestamp ts, x, rnd_double() d from long_sequence(1000000)) timestamp(ts) partition by day]
2023-11-16T13:21:27.6799887Z 2023-11-16T13:21:27.677165Z I i.q.c.CairoEngine locked [table=`tab~`, thread=8548]
2023-11-16T13:21:27.6892533Z 2023-11-16T13:21:27.688421Z I i.q.c.TableWriter open 'tab'
2023-11-16T13:21:27.6934893Z 2023-11-16T13:21:27.692579Z I i.q.c.TableWriter switched partition [path='C:\Users\VSSADM~1\AppData\Local\Temp\junit16836771537375288158\dbRoot\tab~\1970-01-01']
2023-11-16T13:21:27.8031183Z 2023-11-16T13:21:27.791086Z I i.q.c.CairoEngine unlocked [table=`tab`]
2023-11-16T13:21:27.8033216Z 2023-11-16T13:21:27.791183Z I i.q.g.SqlCompilerImpl parse [fd=-1, q=create table if not exists dest as (select x l from long_sequence(10000))]
2023-11-16T13:21:27.8035053Z 2023-11-16T13:21:27.791970Z I i.q.c.CairoEngine locked [table=`dest~`, thread=8548]
2023-11-16T13:21:27.8060748Z 2023-11-16T13:21:27.805601Z I i.q.c.TableWriter open 'dest'
2023-11-16T13:21:27.8079058Z 2023-11-16T13:21:27.807452Z I i.q.c.TableWriter switched partition [path='C:\Users\VSSADM~1\AppData\Local\Temp\junit16836771537375288158\dbRoot\dest~\default']
2023-11-16T13:21:27.8114232Z 2023-11-16T13:21:27.810690Z I i.q.c.CairoEngine unlocked [table=`dest`]
2023-11-16T13:21:27.8167265Z 2023-11-16T13:21:27.812869Z I i.q.g.SqlCompilerImpl parse [fd=5564, q=create table new_tab as (select count(*) from tab t1 join tab t2 on t1.x = t2.x where t1.x > 0)]
2023-11-16T13:21:27.8170283Z 2023-11-16T13:21:27.814364Z I i.q.g.SqlCodeGenerator JIT enabled for (sub)query [tableName=t1, fd=5564]
2023-11-16T13:21:27.8172248Z 2023-11-16T13:21:27.815647Z I i.q.c.CairoEngine locked [table=`new_tab~`, thread=8549]
2023-11-16T13:21:27.8332723Z 2023-11-16T13:21:27.828029Z I i.q.c.TableWriter open 'new_tab'
2023-11-16T13:21:27.8370975Z 2023-11-16T13:21:27.829912Z I i.q.c.TableWriter switched partition [path='C:\Users\VSSADM~1\AppData\Local\Temp\junit16836771537375288158\dbRoot\new_tab~\default']
2023-11-16T13:21:27.8375359Z 2023-11-16T13:21:27.832732Z I i.q.c.TableReader open partition C:\Users\VSSADM~1\AppData\Local\Temp\junit16836771537375288158\dbRoot\tab~\1970-01-01 [rowCount=1000000, partitionNameTxn=-1, transientRowCount=1000000, partitionIndex=0, partitionCount=1]
2023-11-16T13:21:27.8381516Z 2023-11-16T13:21:27.832861Z I pg-server connected [ip=127.0.0.1, fd=17104]
2023-11-16T13:21:27.8383042Z 2023-11-16T13:21:27.832906Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-16T13:21:27.8384444Z 2023-11-16T13:21:27.832927Z I pg-server scheduling disconnect [fd=17104, reason=15]
2023-11-16T13:21:27.8385655Z 2023-11-16T13:21:27.832930Z I pg-server disconnected [ip=127.0.0.1, fd=17104, src=queue]
2023-11-16T13:21:27.8386851Z 2023-11-16T13:21:27.833139Z E i.q.g.SqlCompilerImpl could not create table [error=
2023-11-16T13:21:27.8388013Z io.questdb.cairo.CairoException: [-1] cancelling statement due to user request [fd=5564]
2023-11-16T13:21:27.8389000Z 	at io.questdb.cairo.CairoException.instance(CairoException.java:264)
2023-11-16T13:21:27.8389912Z 	at io.questdb.cairo.CairoException.nonCritical(CairoException.java:114)
2023-11-16T13:21:27.8390860Z 	at io.questdb.cairo.CairoException.queryCancelled(CairoException.java:118)
2023-11-16T13:21:27.8392108Z 	at io.questdb.cairo.sql.NetworkSqlExecutionCircuitBreaker.testTimeout(NetworkSqlExecutionCircuitBreaker.java:189)
2023-11-16T13:21:27.8393860Z 	at io.questdb.cairo.sql.NetworkSqlExecutionCircuitBreaker.statefulThrowExceptionIfTrippedNoThrottle(NetworkSqlExecutionCircuitBreaker.java:175)
2023-11-16T13:21:27.8395731Z 	at io.questdb.cairo.sql.NetworkSqlExecutionCircuitBreaker.statefulThrowExceptionIfTripped(NetworkSqlExecutionCircuitBreaker.java:168)
2023-11-16T13:21:27.8397100Z 	at io.questdb.cairo.TableUtils.populateRowIDHashMap(TableUtils.java:1214)
2023-11-16T13:21:27.8402520Z 	at io.questdb.griffin.engine.join.HashJoinLightRecordCursorFactory$HashJoinRecordCursor.buildMapOfSlaveRecords(HashJoinLightRecordCursorFactory.java:193)
2023-11-16T13:21:27.8404451Z 	at io.questdb.griffin.engine.join.HashJoinLightRecordCursorFactory$HashJoinRecordCursor.size(HashJoinLightRecordCursorFactory.java:176)
2023-11-16T13:21:27.8406135Z 	at io.questdb.griffin.engine.groupby.CountRecordCursorFactory$CountRecordCursor.hasNext(CountRecordCursorFactory.java:97)
2023-11-16T13:21:27.8407443Z 	at io.questdb.griffin.SqlCompilerImpl.copyUnordered(SqlCompilerImpl.java:1764)
2023-11-16T13:21:27.8408460Z 	at io.questdb.griffin.SqlCompilerImpl.copyTableData(SqlCompilerImpl.java:1651)
2023-11-16T13:21:27.8409528Z 	at io.questdb.griffin.SqlCompilerImpl.copyTableDataAndUnlock(SqlCompilerImpl.java:1692)
2023-11-16T13:21:27.8410722Z 	at io.questdb.griffin.SqlCompilerImpl.createTableFromCursorExecutor(SqlCompilerImpl.java:1893)
2023-11-16T13:21:27.8412039Z 	at io.questdb.griffin.SqlCompilerImpl.createTable(SqlCompilerImpl.java:1843)
2023-11-16T13:21:27.8413093Z 	at io.questdb.griffin.SqlCompilerImpl.executeWithRetries(SqlCompilerImpl.java:1944)
2023-11-16T13:21:27.8414204Z 	at io.questdb.griffin.SqlCompilerImpl.createTableWithRetries(SqlCompilerImpl.java:1928)
2023-11-16T13:21:27.8415298Z 	at io.questdb.griffin.SqlCompilerImpl.compileUsingModel(SqlCompilerImpl.java:1445)
2023-11-16T13:21:27.8416343Z 	at io.questdb.griffin.SqlCompilerImpl.compileInner(SqlCompilerImpl.java:1409)
2023-11-16T13:21:27.8417314Z 	at io.questdb.griffin.SqlCompilerImpl.compile(SqlCompilerImpl.java:233)
2023-11-16T13:21:27.8418274Z 	at io.questdb.cairo.pool.SqlCompilerPool$C.compile(SqlCompilerPool.java:98)
2023-11-16T13:21:27.8419354Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.compileQuery(PGConnectionContext.java:1327)
2023-11-16T13:21:27.8420689Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.parseQueryText(PGConnectionContext.java:1927)
2023-11-16T13:21:27.8421953Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.processParse(PGConnectionContext.java:2440)
2023-11-16T13:21:27.8423107Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.parse(PGConnectionContext.java:1862)
2023-11-16T13:21:27.8424308Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.handleClientOperation(PGConnectionContext.java:519)
2023-11-16T13:21:27.8425448Z 	at io.questdb.cutlass.pgwire.PGWireServer$1.lambda$$0(PGWireServer.java:79)
2023-11-16T13:21:27.8426498Z 	at io.questdb.network.AbstractIODispatcher.processIOQueue(AbstractIODispatcher.java:199)
2023-11-16T13:21:27.8427525Z 	at io.questdb.cutlass.pgwire.PGWireServer$1.run(PGWireServer.java:109)
2023-11-16T13:21:27.8428304Z 	at io.questdb.mp.Worker.run(Worker.java:148)
2023-11-16T13:21:27.8428833Z ]
2023-11-16T13:21:27.8429567Z 2023-11-16T13:21:27.833780Z I i.q.c.TableWriter closed 'new_tab'
2023-11-16T13:21:27.8430575Z 2023-11-16T13:21:27.833967Z I i.q.c.CairoEngine unlocked [table=`new_tab`]
2023-11-16T13:21:27.8431910Z 2023-11-16T13:21:27.833973Z E i.q.g.SqlCompilerImpl cancelling statement due to user request [fd=5564] [errno=-1]
2023-11-16T13:21:27.8433319Z 2023-11-16T13:21:27.834362Z I i.q.c.CairoEngine locked [table=`new_tab~`, thread=8549]
2023-11-16T13:21:27.8434471Z 2023-11-16T13:21:27.835978Z I pg-server connected [ip=127.0.0.1, fd=6896]
2023-11-16T13:21:27.8435741Z 2023-11-16T13:21:27.836017Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-16T13:21:27.8437044Z 2023-11-16T13:21:27.836022Z I pg-server scheduling disconnect [fd=6896, reason=15]
2023-11-16T13:21:27.8438237Z 2023-11-16T13:21:27.836025Z I pg-server disconnected [ip=127.0.0.1, fd=6896, src=queue]
2023-11-16T13:21:27.8439518Z 2023-11-16T13:21:27.837534Z I pg-server connected [ip=127.0.0.1, fd=6896]
2023-11-16T13:21:27.8669225Z 2023-11-16T13:21:27.837566Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-16T13:21:27.8672056Z 2023-11-16T13:21:27.837574Z I pg-server scheduling disconnect [fd=6896, reason=15]
2023-11-16T13:21:27.8676648Z 2023-11-16T13:21:27.837577Z I pg-server disconnected [ip=127.0.0.1, fd=6896, src=queue]
2023-11-16T13:21:27.8679590Z 2023-11-16T13:21:27.838148Z I i.q.g.SqlCompilerImpl parse [fd=5564, q=select count(*) from tab t1 join tab t2 on t1.x = t2.x where t1.x > 0]
2023-11-16T13:21:27.8682552Z 2023-11-16T13:21:27.838622Z I i.q.g.SqlCompilerImpl plan [q=`select-group-by count() count from (select [x] from tab t1 timestamp (ts) join select [x] from tab t2 timestamp (ts) on t2.x = t1.x where x > 0) t1`, fd=5564]
2023-11-16T13:21:27.8685174Z 2023-11-16T13:21:27.838896Z I i.q.g.SqlCodeGenerator JIT enabled for (sub)query [tableName=t1, fd=5564]
2023-11-16T13:21:27.8706086Z 2023-11-16T13:21:27.869996Z I pg-server connected [ip=127.0.0.1, fd=18728]
2023-11-16T13:21:27.8709770Z 2023-11-16T13:21:27.870044Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-16T13:21:27.8711239Z 2023-11-16T13:21:27.870058Z I pg-server scheduling disconnect [fd=18728, reason=15]
2023-11-16T13:21:27.8712806Z 2023-11-16T13:21:27.870061Z I pg-server disconnected [ip=127.0.0.1, fd=18728, src=queue]
2023-11-16T13:21:27.8721027Z 2023-11-16T13:21:27.871557Z I pg-server connected [ip=127.0.0.1, fd=11408]
2023-11-16T13:21:28.0747805Z 2023-11-16T13:21:27.871631Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-16T13:21:28.2777964Z 2023-11-16T13:21:27.871640Z I pg-server scheduling disconnect [fd=11408, reason=15]
2023-11-16T13:21:28.4736691Z 2023-11-16T13:21:27.871645Z I pg-server disconnected [ip=127.0.0.1, fd=11408, src=queue]
2023-11-16T13:21:28.5237930Z 2023-11-16T13:21:27.872105Z I i.q.g.SqlCompilerImpl parse [fd=5564, q=insert into dest select count(*)::timestamp, 0, 0.0 from tab t1 join tab t2 on t1.x = t2.x where t1.x > 0]
2023-11-16T13:21:28.6691602Z 2023-11-16T13:21:27.873204Z I i.q.c.p.WriterPool >> [table=`dest~`, thread=8549]
2023-11-16T13:21:28.7014200Z 2023-11-16T13:21:27.873421Z I i.q.g.SqlCodeGenerator JIT enabled for (sub)query [tableName=t1, fd=5564]
2023-11-16T13:21:28.7231214Z 2023-11-16T13:21:28.457408Z I i.q.c.TableReader open partition C:\Users\VSSADM~1\AppData\Local\Temp\junit16836771537375288158\dbRoot\tab~\1970-01-01 [rowCount=1000000, partitionNameTxn=-1, transientRowCount=1000000, partitionIndex=0, partitionCount=1]
2023-11-16T13:21:28.7285810Z Exception in thread "cancellation thread" java.lang.RuntimeException: org.postgresql.util.PSQLException: This connection has been closed.
2023-11-16T13:21:28.7289193Z 	at io.questdb.test.cutlass.pgwire.PGJobContextTest.lambda$testCancelRunningQuery$15(PGJobContextTest.java:2375)
2023-11-16T13:21:28.7291061Z 	at java.base/java.lang.Thread.run(Thread.java:829)
2023-11-16T13:21:28.7359766Z Caused by: org.postgresql.util.PSQLException: This connection has been closed.
2023-11-16T13:21:28.7363721Z 	at org.postgresql.jdbc.PgConnection.checkClosed(PgConnection.java:907)
2023-11-16T13:21:28.7365464Z 	at org.postgresql.jdbc.PgConnection.cancelQuery(PgConnection.java:1119)
2023-11-16T13:21:28.7373300Z 	at io.questdb.test.cutlass.pgwire.PGJobContextTest.lambda$testCancelRunningQuery$15(PGJobContextTest.java:2372)
2023-11-16T13:21:28.7375003Z 	... 1 more
2023-11-16T13:21:28.7376195Z 2023-11-16T13:21:28.669440Z I pg-server connected [ip=127.0.0.1, fd=18752]
2023-11-16T13:21:28.7377715Z 2023-11-16T13:21:28.669490Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-16T13:21:28.7379226Z 2023-11-16T13:21:28.669505Z I pg-server scheduling disconnect [fd=18752, reason=15]
2023-11-16T13:21:28.7380661Z 2023-11-16T13:21:28.669509Z I pg-server disconnected [ip=127.0.0.1, fd=18752, src=queue]
2023-11-16T13:21:28.7382097Z 2023-11-16T13:21:28.699906Z I i.q.c.p.WriterPool << [table=`dest~`, thread=8549]
2023-11-16T13:21:28.7383410Z 2023-11-16T13:21:28.700199Z I pg-server connected [ip=127.0.0.1, fd=18636]
2023-11-16T13:21:28.7384855Z 2023-11-16T13:21:28.700243Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-16T13:21:28.7386354Z 2023-11-16T13:21:28.700249Z I pg-server scheduling disconnect [fd=18636, reason=15]
2023-11-16T13:21:28.7387778Z 2023-11-16T13:21:28.700253Z I pg-server disconnected [ip=127.0.0.1, fd=18636, src=queue]
2023-11-16T13:21:28.7389179Z 2023-11-16T13:21:28.700922Z I pg-server scheduling disconnect [fd=5564, reason=11]
2023-11-16T13:21:28.7390642Z 2023-11-16T13:21:28.700971Z I i.q.t.c.p.PGJobContextTest cleaned worker [name=testing, worker=0]
2023-11-16T13:21:28.7392225Z 2023-11-16T13:21:28.700974Z I i.q.t.c.p.PGJobContextTest os scheduled worker stopped [name=testing_0]
2023-11-16T13:21:28.7395903Z 2023-11-16T13:21:28.705638Z I i.q.t.c.p.PGJobContextTest cleaned worker [name=testing, worker=1]
2023-11-16T13:21:28.7397666Z 2023-11-16T13:21:28.705719Z I i.q.c.p.WriterPool << [table=`sys.text_import_log~`, thread=8548]
2023-11-16T13:21:28.7399176Z 2023-11-16T13:21:28.705745Z I pg-server disconnected [ip=127.0.0.1, fd=5564, src=queue]
2023-11-16T13:21:28.7400630Z 2023-11-16T13:21:28.705927Z I pg-server closed
2023-11-16T13:21:28.7401888Z 2023-11-16T13:21:28.706484Z I i.q.c.p.ReaderPool closed 'tab~' [at=0:0, reason=POOL_CLOSED]
2023-11-16T13:21:28.7403420Z 2023-11-16T13:21:28.707044Z I i.q.c.p.ReaderPool closed 'tab~' [at=0:1, reason=POOL_CLOSED]
2023-11-16T13:21:28.7404818Z 2023-11-16T13:21:28.709223Z I i.q.c.TableWriter closed 'sys.text_import_log'
2023-11-16T13:21:28.7406422Z 2023-11-16T13:21:28.709226Z I i.q.c.p.WriterPool closed [table=`sys.text_import_log~`, reason=POOL_CLOSED, by=8548]
2023-11-16T13:21:28.7407858Z 2023-11-16T13:21:28.709740Z I i.q.c.TableWriter closed 'dest'
2023-11-16T13:21:28.7409173Z 2023-11-16T13:21:28.709743Z I i.q.c.p.WriterPool closed [table=`dest~`, reason=POOL_CLOSED, by=8548]
2023-11-16T13:21:28.7410493Z 2023-11-16T13:21:28.710820Z I i.q.c.TableWriter closed 'tab'
2023-11-16T13:21:28.7411923Z 2023-11-16T13:21:28.710823Z I i.q.c.p.WriterPool closed [table=`tab~`, reason=POOL_CLOSED, by=8548]
2023-11-16T13:21:28.7413619Z 2023-11-16T13:21:28.710871Z I i.q.t.AbstractCairoTest Tearing down test PGJobContextTest#testCancelRunningQuery[NO_WAL]
2023-11-16T13:21:28.7415359Z 2023-11-16T13:21:28.710991Z I i.q.t.AbstractTest Finished test PGJobContextTest#testCancelRunningQuery[NO_WAL]
2023-11-16T13:21:28.7417099Z 2023-11-16T13:21:28.718663Z I i.q.t.AbstractCairoTest Tearing down test PGJobContextTest#testCancelRunningQuery[NO_WAL]
2023-11-16T13:21:28.7418827Z 2023-11-16T13:21:28.718717Z I i.q.t.AbstractTest Finished test PGJobContextTest#testCancelRunningQuery[NO_WAL]
2023-11-16T13:21:28.7420444Z 2023-11-16T13:21:28.718874Z I i.q.t.c.p.PGJobContextTest os scheduled worker stopped [name=testing_1]
2023-11-16T13:21:28.7421768Z random seeds: 1738434582000L, 1700140888726L
2023-11-16T13:21:28.7423558Z 2023-11-16T13:21:28.729979Z E i.q.t.TestListener ***** Test Failed *****io.questdb.test.cutlass.pgwire.PGJobContextTest.testCancelRunningQuery[NO_WAL] duration_ms=1112 : 
2023-11-16T13:21:28.7425893Z java.lang.AssertionError: expected PSQLException with cancel message for query: insert into dest select count(*)::timestamp, 0, 0.0 from tab t1 join tab t2 on t1.x = t2.x where t1.x > 0
2023-11-16T13:21:28.7427493Z 	at org.junit.Assert.fail(Assert.java:89)
2023-11-16T13:21:28.7428693Z 	at io.questdb.test.cutlass.pgwire.PGJobContextTest.lambda$testCancelRunningQuery$16(PGJobContextTest.java:2382)
2023-11-16T13:21:28.7430295Z 	at io.questdb.test.cutlass.pgwire.PGJobContextTest.lambda$assertWithPgServer$169(PGJobContextTest.java:9800)
2023-11-16T13:21:28.7431753Z 	at io.questdb.test.AbstractCairoTest.lambda$assertMemoryLeak$7(AbstractCairoTest.java:943)
2023-11-16T13:21:28.7432994Z 	at io.questdb.test.tools.TestUtils.assertMemoryLeak(TestUtils.java:610)
2023-11-16T13:21:28.7434179Z 	at io.questdb.test.AbstractCairoTest.assertMemoryLeak(AbstractCairoTest.java:940)
2023-11-16T13:21:28.7435411Z 	at io.questdb.test.AbstractCairoTest.assertMemoryLeak(AbstractCairoTest.java:934)
2023-11-16T13:21:28.7436752Z 	at io.questdb.test.cutlass.pgwire.PGJobContextTest.assertWithPgServer(PGJobContextTest.java:9793)
2023-11-16T13:21:28.7438191Z 	at io.questdb.test.cutlass.pgwire.PGJobContextTest.assertWithPgServer(PGJobContextTest.java:9756)
2023-11-16T13:21:28.7439624Z 	at io.questdb.test.cutlass.pgwire.PGJobContextTest.assertWithPgServer(PGJobContextTest.java:9741)
2023-11-16T13:21:28.7441092Z 	at io.questdb.test.cutlass.pgwire.PGJobContextTest.testCancelRunningQuery(PGJobContextTest.java:2356)
2023-11-16T13:21:28.7442372Z 	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2023-11-16T13:21:28.7443586Z 	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2023-11-16T13:21:28.7444970Z 	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2023-11-16T13:21:28.7446151Z 	at java.lang.reflect.Method.invoke(Method.java:566)
2023-11-16T13:21:28.7447232Z 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
2023-11-16T13:21:28.7448680Z 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
2023-11-16T13:21:28.7449970Z 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
2023-11-16T13:21:28.7451242Z 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
2023-11-16T13:21:28.7452479Z 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
2023-11-16T13:21:28.7453678Z 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
2023-11-16T13:21:28.7454793Z 	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
2023-11-16T13:21:28.7456029Z 	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:299)
2023-11-16T13:21:28.7457621Z 	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:293)
2023-11-16T13:21:28.7458951Z 	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
2023-11-16T13:21:28.7459822Z 	at java.lang.Thread.run(Thread.java:829)
2023-11-16T13:21:28.7461409Z 2023-11-16T13:21:28.730094Z I i.q.t.TestListener <<<< io.questdb.test.cutlass.pgwire.PGJobContextTest.testCancelRunningQuery[NO_WAL] duration_ms=1112
```

another one of these.
log is very similar to the above:
```
2023-11-24T11:49:11.7174115Z 2023-11-24T11:49:11.710981Z I i.q.t.TestListener >>>> io.questdb.test.cutlass.pgwire.PGJobContextTest.testCancelRunningQuery[WITH_WAL]
2023-11-24T11:49:11.7174673Z 2023-11-24T11:49:11.711013Z I i.q.t.c.p.PGJobContextTest random seeds: 1481402264700L, 1700826551708L
2023-11-24T11:49:11.7180038Z 2023-11-24T11:49:11.712932Z I i.q.t.AbstractTest Starting test PGJobContextTest#testCancelRunningQuery[WITH_WAL]
2023-11-24T11:49:11.7187234Z 2023-11-24T11:49:11.715204Z I i.q.t.s.TestFilesFacadeImpl cannot remove file: C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\tables.d.0, errno:2
2023-11-24T11:49:11.7189345Z 2023-11-24T11:49:11.718124Z I i.q.c.TableNameRegistryStore reloading tables file [path=C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\tables.d.0]
2023-11-24T11:49:11.7190939Z 2023-11-24T11:49:11.718263Z I i.q.t.c.p.PGJobContextTest fragmentation params [sendBufferSize=2048, forceSendFragmentationChunkSize=371, recvBufferSize=7680, forceRecvFragmentationChunkSize=265]
2023-11-24T11:49:11.7196053Z 2023-11-24T11:49:11.719224Z I i.q.t.c.p.PGJobContextTest asserting PG Wire server [mode=EXTENDED, binary=true, prepareThreshold=-2]
2023-11-24T11:49:11.7197632Z 2023-11-24T11:49:11.719231Z I i.q.t.AbstractTest Starting test PGJobContextTest#testCancelRunningQuery[WITH_WAL]
2023-11-24T11:49:11.7221700Z 2023-11-24T11:49:11.721877Z I i.q.c.TableNameRegistryStore reloading tables file [path=C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\tables.d.0]
2023-11-24T11:49:11.7227166Z 2023-11-24T11:49:11.722413Z I i.q.g.SqlCompilerImpl parse [fd=-1, q=CREATE TABLE IF NOT EXISTS "sys.text_import_log" (ts timestamp, id string, table_name symbol, file symbol, phase symbol, status symbol, message string,rows_handled long,rows_imported long,errors long) timestamp(ts) partition by DAY BYPASS WAL]
2023-11-24T11:49:11.7237039Z 2023-11-24T11:49:11.723390Z I i.q.c.CairoEngine locked [table=`sys.text_import_log~`, thread=7267]
2023-11-24T11:49:11.7720477Z 2023-11-24T11:49:11.769233Z I i.q.c.p.WriterPool created [table=`sys.text_import_log~`, thread=7267]
2023-11-24T11:49:11.7722620Z 2023-11-24T11:49:11.769263Z I i.q.c.TableWriter open 'sys.text_import_log'
2023-11-24T11:49:11.7874399Z 2023-11-24T11:49:11.781014Z I i.q.c.CairoEngine unlocked [table=`sys.text_import_log`]
2023-11-24T11:49:11.7875200Z 2023-11-24T11:49:11.781045Z I i.q.c.p.WriterPool >> [table=`sys.text_import_log~`, thread=7267]
2023-11-24T11:49:11.7875843Z 2023-11-24T11:49:11.781411Z A pg-server listening on 0.0.0.0:0 [fd=16884 backlog=64]
2023-11-24T11:49:11.7876365Z 2023-11-24T11:49:11.781693Z I i.q.t.c.p.PGJobContextTest worker pool started [pool=testing]
2023-11-24T11:49:11.7876905Z 2023-11-24T11:49:11.783319Z I i.q.t.c.p.PGJobContextTest os scheduled worker started [name=testing_0]
2023-11-24T11:49:11.7877437Z 2023-11-24T11:49:11.783512Z I i.q.t.c.p.PGJobContextTest os scheduled worker started [name=testing_1]
2023-11-24T11:49:11.7877938Z 2023-11-24T11:49:11.783611Z I pg-server connected [ip=127.0.0.1, fd=7136]
2023-11-24T11:49:11.7878456Z 2023-11-24T11:49:11.784150Z I i.q.g.SqlCompilerImpl parse [fd=7136, q=SET extra_float_digits = 3]
2023-11-24T11:49:11.7878992Z 2023-11-24T11:49:11.784302Z I i.q.g.SqlCompilerImpl parse [fd=7136, q=SET application_name = 'PostgreSQL JDBC Driver']
2023-11-24T11:49:11.7879724Z 2023-11-24T11:49:11.784539Z I i.q.g.SqlCompilerImpl parse [fd=-1, q=create table if not exists tab as (select x::timestamp ts, x, rnd_double() d from long_sequence(1000000)) timestamp(ts) partition by day]
2023-11-24T11:49:11.7880317Z 2023-11-24T11:49:11.785527Z I i.q.c.CairoEngine locked [table=`tab~2`, thread=7267]
2023-11-24T11:49:11.8177274Z 2023-11-24T11:49:11.816749Z I i.q.c.p.WriterPool created [table=`tab~2`, thread=7267]
2023-11-24T11:49:11.8180322Z 2023-11-24T11:49:11.816778Z I i.q.c.TableWriter open 'tab'
2023-11-24T11:49:11.8213913Z 2023-11-24T11:49:11.821051Z I i.q.c.CairoEngine unlocked [table=`tab`]
2023-11-24T11:49:11.8232240Z 2023-11-24T11:49:11.822893Z I i.q.c.w.WalWriter open 'tab~2'
2023-11-24T11:49:11.8270031Z 2023-11-24T11:49:11.826652Z I i.q.c.w.WalWriter opened WAL segment [path='C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\tab~2\wal1\0\_event.i']
2023-11-24T11:49:11.9601181Z 2023-11-24T11:49:11.951508Z I i.q.c.w.WalWriter committed data block [wal=C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\tab~2\wal1\0, seqTxn=1, rowLo=0, roHi=1000000, minTimestamp=1970-01-01T00:00:00.000001Z, maxTimestamp=1970-01-01T00:00:01.000000Z]
2023-11-24T11:49:11.9603461Z 2023-11-24T11:49:11.951644Z I i.q.g.SqlCompilerImpl parse [fd=-1, q=create table if not exists dest as (select x l from long_sequence(10000))]
2023-11-24T11:49:11.9604960Z 2023-11-24T11:49:11.952695Z I i.q.c.CairoEngine locked [table=`dest~`, thread=7267]
2023-11-24T11:49:11.9794113Z 2023-11-24T11:49:11.970967Z I i.q.c.TableWriter open 'dest'
2023-11-24T11:49:11.9795519Z 2023-11-24T11:49:11.973685Z I i.q.c.TableWriter switched partition [path='C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\dest~\default']
2023-11-24T11:49:11.9889028Z 2023-11-24T11:49:11.981399Z I i.q.c.CairoEngine unlocked [table=`dest`]
2023-11-24T11:49:11.9890387Z 2023-11-24T11:49:11.981495Z I i.q.c.p.WriterPool >> [table=`tab~2`, thread=7267]
2023-11-24T11:49:11.9891823Z 2023-11-24T11:49:11.982182Z I i.q.c.TableWriter processing WAL [path=C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\tab~2\wal1\0, roLo=0, roHi=1000000, seqTxn=1, tsMin=1970-01-01T00:00:00.000001Z, tsMax=1970-01-01T00:00:01.000000Z, commitToTimestamp=294247-01-10T04:00:54.775807Z]
2023-11-24T11:49:11.9893239Z 2023-11-24T11:49:11.983628Z I i.q.c.TableWriter switched partition [path='C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\tab~2\1970-01-01']
2023-11-24T11:49:11.9895148Z 2023-11-24T11:49:11.984692Z I i.q.c.TableWriter o3 partition task [table=tab, partitionIsReadOnly=false, srcOooBatchRowSize=1000000, srcOooLo=0, srcOooHi=999999, srcOooMax=1000000, o3RowCount=1000000, o3LagRowCount=0, srcDataMax=0, o3TimestampMin=1970-01-01T00:00:00.000001Z, o3Timestamp=1970-01-01T00:00:00.000001Z, o3TimestampMax=1970-01-01T00:00:01.000000Z, partitionTimestamp=1970-01-01T00:00:00.000000Z, partitionIndex=-1, newPartitionSize=1000000, maxTimestamp=, last=false, append=false, pCount=1, flattenTimestamp=false, memUsed=52629410]
2023-11-24T11:49:12.0261214Z 2023-11-24T11:49:12.025438Z I i.q.c.TableWriter switched partition [path='C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\tab~2\1970-01-01']
2023-11-24T11:49:12.0290327Z 2023-11-24T11:49:12.028452Z I i.q.c.w.ApplyWal2TableJob job finished [table=tab~2, seqTxn=1, transactions=1, rows=1000000, time=46ms, rate=21527134rows/s, physicalWrittenRowsMultiplier=1.0]
2023-11-24T11:49:12.0291655Z 2023-11-24T11:49:12.028494Z I i.q.c.p.WriterPool << [table=`tab~2`, thread=7267]
2023-11-24T11:49:12.0523154Z 2023-11-24T11:49:12.046685Z I pg-server connected [ip=127.0.0.1, fd=17064]
2023-11-24T11:49:12.0527020Z 2023-11-24T11:49:12.046783Z I i.q.g.SqlCompilerImpl parse [fd=7136, q=create table new_tab as (select count(*) from tab t1 join tab t2 on t1.x = t2.x where t1.x > 0)]
2023-11-24T11:49:12.0531873Z 2023-11-24T11:49:12.046802Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0541903Z 2023-11-24T11:49:12.046940Z I pg-server scheduling disconnect [fd=17064, reason=15]
2023-11-24T11:49:12.0543697Z 2023-11-24T11:49:12.046948Z I pg-server disconnected [ip=127.0.0.1, fd=17064, src=queue]
2023-11-24T11:49:12.0545286Z 2023-11-24T11:49:12.051500Z I pg-server connected [ip=127.0.0.1, fd=10960]
2023-11-24T11:49:12.0546768Z 2023-11-24T11:49:12.051727Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0548379Z 2023-11-24T11:49:12.051767Z I pg-server scheduling disconnect [fd=10960, reason=15]
2023-11-24T11:49:12.0549894Z 2023-11-24T11:49:12.051771Z I pg-server disconnected [ip=127.0.0.1, fd=10960, src=queue]
2023-11-24T11:49:12.0569853Z 2023-11-24T11:49:12.052443Z I i.q.g.SqlCodeGenerator JIT enabled for (sub)query [tableName=t1, fd=7136]
2023-11-24T11:49:12.0572651Z 2023-11-24T11:49:12.053608Z I pg-server connected [ip=127.0.0.1, fd=16388]
2023-11-24T11:49:12.0573414Z 2023-11-24T11:49:12.053675Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0574007Z 2023-11-24T11:49:12.053685Z I pg-server scheduling disconnect [fd=16388, reason=15]
2023-11-24T11:49:12.0574560Z 2023-11-24T11:49:12.053689Z I pg-server disconnected [ip=127.0.0.1, fd=16388, src=queue]
2023-11-24T11:49:12.0575085Z 2023-11-24T11:49:12.055168Z I i.q.c.CairoEngine locked [table=`new_tab~`, thread=7268]
2023-11-24T11:49:12.0575852Z 2023-11-24T11:49:12.056010Z I pg-server connected [ip=127.0.0.1, fd=16616]
2023-11-24T11:49:12.0576420Z 2023-11-24T11:49:12.056055Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0576980Z 2023-11-24T11:49:12.056068Z I pg-server scheduling disconnect [fd=16616, reason=15]
2023-11-24T11:49:12.0577541Z 2023-11-24T11:49:12.056072Z I pg-server disconnected [ip=127.0.0.1, fd=16616, src=queue]
2023-11-24T11:49:12.0590547Z 2023-11-24T11:49:12.058574Z I pg-server connected [ip=127.0.0.1, fd=16616]
2023-11-24T11:49:12.0591398Z 2023-11-24T11:49:12.058616Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0592011Z 2023-11-24T11:49:12.058628Z I pg-server scheduling disconnect [fd=16616, reason=15]
2023-11-24T11:49:12.0592754Z 2023-11-24T11:49:12.058655Z I pg-server disconnected [ip=127.0.0.1, fd=16616, src=queue]
2023-11-24T11:49:12.0615914Z 2023-11-24T11:49:12.061153Z I pg-server connected [ip=127.0.0.1, fd=9744]
2023-11-24T11:49:12.0617110Z 2023-11-24T11:49:12.061194Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0618125Z 2023-11-24T11:49:12.061205Z I pg-server scheduling disconnect [fd=9744, reason=15]
2023-11-24T11:49:12.0618811Z 2023-11-24T11:49:12.061231Z I pg-server disconnected [ip=127.0.0.1, fd=9744, src=queue]
2023-11-24T11:49:12.0644250Z 2023-11-24T11:49:12.063783Z I pg-server connected [ip=127.0.0.1, fd=6568]
2023-11-24T11:49:12.0645375Z 2023-11-24T11:49:12.063874Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0646102Z 2023-11-24T11:49:12.064033Z I pg-server scheduling disconnect [fd=6568, reason=15]
2023-11-24T11:49:12.0647022Z 2023-11-24T11:49:12.064040Z I pg-server disconnected [ip=127.0.0.1, fd=6568, src=queue]
2023-11-24T11:49:12.0670678Z 2023-11-24T11:49:12.066565Z I pg-server connected [ip=127.0.0.1, fd=6568]
2023-11-24T11:49:12.0671434Z 2023-11-24T11:49:12.066636Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0672183Z 2023-11-24T11:49:12.066666Z I pg-server scheduling disconnect [fd=6568, reason=15]
2023-11-24T11:49:12.0672702Z 2023-11-24T11:49:12.066671Z I pg-server disconnected [ip=127.0.0.1, fd=6568, src=queue]
2023-11-24T11:49:12.0696528Z 2023-11-24T11:49:12.069235Z I pg-server connected [ip=127.0.0.1, fd=6568]
2023-11-24T11:49:12.0697291Z 2023-11-24T11:49:12.069294Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0697869Z 2023-11-24T11:49:12.069320Z I pg-server scheduling disconnect [fd=6568, reason=15]
2023-11-24T11:49:12.0699324Z 2023-11-24T11:49:12.069326Z I pg-server disconnected [ip=127.0.0.1, fd=6568, src=queue]
2023-11-24T11:49:12.0723351Z 2023-11-24T11:49:12.071806Z I pg-server connected [ip=127.0.0.1, fd=6568]
2023-11-24T11:49:12.0724373Z 2023-11-24T11:49:12.071847Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0725096Z 2023-11-24T11:49:12.071898Z I pg-server scheduling disconnect [fd=6568, reason=15]
2023-11-24T11:49:12.0725893Z 2023-11-24T11:49:12.071904Z I pg-server disconnected [ip=127.0.0.1, fd=6568, src=queue]
2023-11-24T11:49:12.0736941Z 2023-11-24T11:49:12.073434Z I i.q.c.TableWriter open 'new_tab'
2023-11-24T11:49:12.0751897Z 2023-11-24T11:49:12.074568Z I pg-server connected [ip=127.0.0.1, fd=9744]
2023-11-24T11:49:12.0753407Z 2023-11-24T11:49:12.074610Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0754980Z 2023-11-24T11:49:12.074621Z I pg-server scheduling disconnect [fd=9744, reason=15]
2023-11-24T11:49:12.0756189Z 2023-11-24T11:49:12.074625Z I pg-server disconnected [ip=127.0.0.1, fd=9744, src=queue]
2023-11-24T11:49:12.0768628Z 2023-11-24T11:49:12.076199Z I pg-server connected [ip=127.0.0.1, fd=16616]
2023-11-24T11:49:12.0769663Z 2023-11-24T11:49:12.076238Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0770564Z 2023-11-24T11:49:12.076248Z I pg-server scheduling disconnect [fd=16616, reason=15]
2023-11-24T11:49:12.0771125Z 2023-11-24T11:49:12.076251Z I pg-server disconnected [ip=127.0.0.1, fd=16616, src=queue]
2023-11-24T11:49:12.0771916Z 2023-11-24T11:49:12.076294Z I i.q.c.TableWriter switched partition [path='C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\new_tab~\default']
2023-11-24T11:49:12.0804927Z 2023-11-24T11:49:12.078802Z I pg-server connected [ip=127.0.0.1, fd=9388]
2023-11-24T11:49:12.0834152Z 2023-11-24T11:49:12.078888Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0840909Z 2023-11-24T11:49:12.078898Z I pg-server scheduling disconnect [fd=9388, reason=15]
2023-11-24T11:49:12.0842221Z 2023-11-24T11:49:12.078901Z I pg-server disconnected [ip=127.0.0.1, fd=9388, src=queue]
2023-11-24T11:49:12.0843425Z 2023-11-24T11:49:12.079471Z I i.q.c.TableReader open partition C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\tab~2\1970-01-01 [rowCount=1000000, partitionNameTxn=-1, transientRowCount=1000000, partitionIndex=0, partitionCount=1]
2023-11-24T11:49:12.0844162Z 2023-11-24T11:49:12.079881Z E i.q.g.SqlCompilerImpl could not create table [error=
2023-11-24T11:49:12.0844699Z io.questdb.cairo.CairoException: [-1] cancelling statement due to user request [fd=7136]
2023-11-24T11:49:12.0845095Z 	at io.questdb.cairo.CairoException.instance(CairoException.java:269)
2023-11-24T11:49:12.0845480Z 	at io.questdb.cairo.CairoException.nonCritical(CairoException.java:114)
2023-11-24T11:49:12.0845877Z 	at io.questdb.cairo.CairoException.queryCancelled(CairoException.java:118)
2023-11-24T11:49:12.0846317Z 	at io.questdb.cairo.sql.NetworkSqlExecutionCircuitBreaker.testTimeout(NetworkSqlExecutionCircuitBreaker.java:189)
2023-11-24T11:49:12.0846822Z 	at io.questdb.cairo.sql.NetworkSqlExecutionCircuitBreaker.statefulThrowExceptionIfTrippedNoThrottle(NetworkSqlExecutionCircuitBreaker.java:175)
2023-11-24T11:49:12.0847355Z 	at io.questdb.cairo.sql.NetworkSqlExecutionCircuitBreaker.statefulThrowExceptionIfTripped(NetworkSqlExecutionCircuitBreaker.java:168)
2023-11-24T11:49:12.0847791Z 	at io.questdb.cairo.TableUtils.populateRowIDHashMap(TableUtils.java:1214)
2023-11-24T11:49:12.0848266Z 	at io.questdb.griffin.engine.join.HashJoinLightRecordCursorFactory$HashJoinRecordCursor.buildMapOfSlaveRecords(HashJoinLightRecordCursorFactory.java:193)
2023-11-24T11:49:12.0848997Z 	at io.questdb.griffin.engine.join.HashJoinLightRecordCursorFactory$HashJoinRecordCursor.size(HashJoinLightRecordCursorFactory.java:176)
2023-11-24T11:49:12.0849523Z 	at io.questdb.griffin.engine.groupby.CountRecordCursorFactory$CountRecordCursor.hasNext(CountRecordCursorFactory.java:109)
2023-11-24T11:49:12.0849981Z 	at io.questdb.griffin.SqlCompilerImpl.copyUnordered(SqlCompilerImpl.java:1763)
2023-11-24T11:49:12.0850560Z 	at io.questdb.griffin.SqlCompilerImpl.copyTableData(SqlCompilerImpl.java:1647)
2023-11-24T11:49:12.0851003Z 	at io.questdb.griffin.SqlCompilerImpl.copyTableDataAndUnlock(SqlCompilerImpl.java:1688)
2023-11-24T11:49:12.0851468Z 	at io.questdb.griffin.SqlCompilerImpl.createTableFromCursorExecutor(SqlCompilerImpl.java:1892)
2023-11-24T11:49:12.0852262Z 	at io.questdb.griffin.SqlCompilerImpl.createTable(SqlCompilerImpl.java:1842)
2023-11-24T11:49:12.0852810Z 	at io.questdb.griffin.SqlCompilerImpl.executeWithRetries(SqlCompilerImpl.java:1943)
2023-11-24T11:49:12.0853758Z 	at io.questdb.griffin.SqlCompilerImpl.createTableWithRetries(SqlCompilerImpl.java:1927)
2023-11-24T11:49:12.0854812Z 	at io.questdb.griffin.SqlCompilerImpl.compileUsingModel(SqlCompilerImpl.java:1441)
2023-11-24T11:49:12.0857786Z 	at io.questdb.griffin.SqlCompilerImpl.compileInner(SqlCompilerImpl.java:1405)
2023-11-24T11:49:12.0865724Z 	at io.questdb.griffin.SqlCompilerImpl.compile(SqlCompilerImpl.java:229)
2023-11-24T11:49:12.0869073Z 	at io.questdb.cairo.pool.SqlCompilerPool$C.compile(SqlCompilerPool.java:98)
2023-11-24T11:49:12.0870745Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.compileQuery(PGConnectionContext.java:1332)
2023-11-24T11:49:12.0873378Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.parseQueryText(PGConnectionContext.java:1932)
2023-11-24T11:49:12.0876205Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.processParse(PGConnectionContext.java:2445)
2023-11-24T11:49:12.0877470Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.parse(PGConnectionContext.java:1867)
2023-11-24T11:49:12.0880155Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.handleClientOperation(PGConnectionContext.java:526)
2023-11-24T11:49:12.0881268Z 	at io.questdb.cutlass.pgwire.PGWireServer$1.lambda$$0(PGWireServer.java:79)
2023-11-24T11:49:12.0882239Z 	at io.questdb.network.AbstractIODispatcher.processIOQueue(AbstractIODispatcher.java:199)
2023-11-24T11:49:12.0883700Z 	at io.questdb.cutlass.pgwire.PGWireServer$1.run(PGWireServer.java:109)
2023-11-24T11:49:12.0891622Z 	at io.questdb.mp.Worker.run(Worker.java:148)
2023-11-24T11:49:12.0897306Z ]
2023-11-24T11:49:12.0898269Z 2023-11-24T11:49:12.080805Z I i.q.c.TableWriter closed 'new_tab'
2023-11-24T11:49:12.0899045Z 2023-11-24T11:49:12.081066Z I i.q.c.CairoEngine unlocked [table=`new_tab`]
2023-11-24T11:49:12.0899623Z 2023-11-24T11:49:12.081094Z E i.q.g.SqlCompilerImpl cancelling statement due to user request [fd=7136] [errno=-1]
2023-11-24T11:49:12.0900160Z 2023-11-24T11:49:12.081463Z I i.q.c.CairoEngine locked [table=`new_tab~`, thread=7268]
2023-11-24T11:49:12.0900803Z 2023-11-24T11:49:12.083082Z I pg-server connected [ip=127.0.0.1, fd=1784]
2023-11-24T11:49:12.0901350Z 2023-11-24T11:49:12.083126Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0902033Z 2023-11-24T11:49:12.083138Z I pg-server scheduling disconnect [fd=1784, reason=15]
2023-11-24T11:49:12.0902540Z 2023-11-24T11:49:12.083142Z I pg-server disconnected [ip=127.0.0.1, fd=1784, src=queue]
2023-11-24T11:49:12.0903035Z 2023-11-24T11:49:12.085748Z I pg-server connected [ip=127.0.0.1, fd=16016]
2023-11-24T11:49:12.0903544Z 2023-11-24T11:49:12.085790Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0904056Z 2023-11-24T11:49:12.085802Z I pg-server scheduling disconnect [fd=16016, reason=15]
2023-11-24T11:49:12.0904564Z 2023-11-24T11:49:12.085806Z I pg-server disconnected [ip=127.0.0.1, fd=16016, src=queue]
2023-11-24T11:49:12.0905127Z 2023-11-24T11:49:12.086475Z I i.q.g.SqlCompilerImpl parse [fd=7136, q=select count(*) from tab t1 join tab t2 on t1.x = t2.x where t1.x > 0]
2023-11-24T11:49:12.0905837Z 2023-11-24T11:49:12.087296Z I i.q.g.SqlCompilerImpl plan [q=`select-group-by count() count from (select [x] from tab t1 timestamp (ts) join select [x] from tab t2 timestamp (ts) on t2.x = t1.x where x > 0) t1`, fd=7136]
2023-11-24T11:49:12.0906459Z 2023-11-24T11:49:12.087729Z I i.q.g.SqlCodeGenerator JIT enabled for (sub)query [tableName=t1, fd=7136]
2023-11-24T11:49:12.0906953Z 2023-11-24T11:49:12.088896Z I pg-server connected [ip=127.0.0.1, fd=17260]
2023-11-24T11:49:12.0907454Z 2023-11-24T11:49:12.088935Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0908150Z 2023-11-24T11:49:12.088946Z I pg-server scheduling disconnect [fd=17260, reason=15]
2023-11-24T11:49:12.0908852Z 2023-11-24T11:49:12.088949Z I pg-server disconnected [ip=127.0.0.1, fd=17260, src=queue]
2023-11-24T11:49:12.0920213Z 2023-11-24T11:49:12.091452Z I pg-server connected [ip=127.0.0.1, fd=12236]
2023-11-24T11:49:12.0924839Z 2023-11-24T11:49:12.091511Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0926243Z 2023-11-24T11:49:12.091537Z I pg-server scheduling disconnect [fd=12236, reason=15]
2023-11-24T11:49:12.0929021Z 2023-11-24T11:49:12.091543Z I pg-server disconnected [ip=127.0.0.1, fd=12236, src=queue]
2023-11-24T11:49:12.0931393Z 2023-11-24T11:49:12.092254Z I i.q.g.SqlCompilerImpl parse [fd=7136, q=insert into dest select count(*)::timestamp, 0, 0.0 from tab t1 join tab t2 on t1.x = t2.x where t1.x > 0]
2023-11-24T11:49:12.0939829Z 2023-11-24T11:49:12.093694Z I i.q.c.p.WriterPool >> [table=`dest~`, thread=7269]
2023-11-24T11:49:12.0943047Z 2023-11-24T11:49:12.093986Z I i.q.g.SqlCodeGenerator JIT enabled for (sub)query [tableName=t1, fd=7136]
2023-11-24T11:49:12.0961163Z 2023-11-24T11:49:12.095388Z I pg-server connected [ip=127.0.0.1, fd=15936]
2023-11-24T11:49:12.0962101Z 2023-11-24T11:49:12.095426Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0962650Z 2023-11-24T11:49:12.095437Z I pg-server scheduling disconnect [fd=15936, reason=15]
2023-11-24T11:49:12.0963185Z 2023-11-24T11:49:12.095440Z I pg-server disconnected [ip=127.0.0.1, fd=15936, src=queue]
2023-11-24T11:49:12.0963682Z 2023-11-24T11:49:12.095838Z I i.q.c.p.WriterPool << [table=`dest~`, thread=7269]
2023-11-24T11:49:12.0988690Z 2023-11-24T11:49:12.098087Z I pg-server connected [ip=127.0.0.1, fd=10960]
2023-11-24T11:49:12.0993519Z 2023-11-24T11:49:12.098129Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.0995296Z 2023-11-24T11:49:12.098154Z I pg-server scheduling disconnect [fd=10960, reason=15]
2023-11-24T11:49:12.0998788Z 2023-11-24T11:49:12.098160Z I pg-server disconnected [ip=127.0.0.1, fd=10960, src=queue]
2023-11-24T11:49:12.1000311Z 2023-11-24T11:49:12.098780Z I i.q.g.SqlCompilerImpl parse [fd=7136, q=update dest 
2023-11-24T11:49:12.1001363Z set l = t1.x 
2023-11-24T11:49:12.1002433Z from (tab where d > 0  limit 1, -1 ) t1 
2023-11-24T11:49:12.1003288Z where 
2023-11-24T11:49:12.1004271Z 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA' || t1.x = 
2023-11-24T11:49:12.1017999Z 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA' || dest.l || '00000']
2023-11-24T11:49:12.1055442Z 2023-11-24T11:49:12.102202Z I i.q.g.SqlCodeGenerator JIT enabled for (sub)query [tableName=tab, fd=7136]
2023-11-24T11:49:12.1069530Z 2023-11-24T11:49:12.102572Z I pg-server connected [ip=127.0.0.1, fd=9512]
2023-11-24T11:49:12.1070875Z 2023-11-24T11:49:12.102610Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.1072071Z 2023-11-24T11:49:12.102621Z I pg-server scheduling disconnect [fd=9512, reason=15]
2023-11-24T11:49:12.1073202Z 2023-11-24T11:49:12.102624Z I pg-server disconnected [ip=127.0.0.1, fd=9512, src=queue]
2023-11-24T11:49:12.1074701Z 2023-11-24T11:49:12.103449Z I i.q.c.p.WriterPool >> [table=`dest~`, thread=7269]
2023-11-24T11:49:12.1075861Z 2023-11-24T11:49:12.103474Z I i.q.g.UpdateOperatorImpl updating [table=dest instance=-1]
2023-11-24T11:49:12.1077418Z 2023-11-24T11:49:12.103693Z I i.q.c.TableReader open partition C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\dest~\default [rowCount=10000, partitionNameTxn=-1, transientRowCount=10000, partitionIndex=0, partitionCount=1]
2023-11-24T11:49:12.1078781Z 2023-11-24T11:49:12.105683Z I pg-server connected [ip=127.0.0.1, fd=17112]
2023-11-24T11:49:12.1095211Z 2023-11-24T11:49:12.105725Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.1096571Z 2023-11-24T11:49:12.105736Z I pg-server scheduling disconnect [fd=17112, reason=15]
2023-11-24T11:49:12.1097960Z 2023-11-24T11:49:12.105740Z I pg-server disconnected [ip=127.0.0.1, fd=17112, src=queue]
2023-11-24T11:49:12.1099143Z 2023-11-24T11:49:12.108413Z I pg-server connected [ip=127.0.0.1, fd=9780]
2023-11-24T11:49:12.1100261Z 2023-11-24T11:49:12.108454Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.1101384Z 2023-11-24T11:49:12.108465Z I pg-server scheduling disconnect [fd=9780, reason=15]
2023-11-24T11:49:12.1102520Z 2023-11-24T11:49:12.108468Z I pg-server disconnected [ip=127.0.0.1, fd=9780, src=queue]
2023-11-24T11:49:12.1122016Z 2023-11-24T11:49:12.111189Z I pg-server connected [ip=127.0.0.1, fd=9780]
2023-11-24T11:49:12.1123566Z 2023-11-24T11:49:12.111227Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.1140900Z 2023-11-24T11:49:12.111237Z I pg-server scheduling disconnect [fd=9780, reason=15]
2023-11-24T11:49:12.1141823Z 2023-11-24T11:49:12.111240Z I pg-server disconnected [ip=127.0.0.1, fd=9780, src=queue]
2023-11-24T11:49:12.1142421Z 2023-11-24T11:49:12.112487Z E i.q.g.UpdateOperatorImpl could not update
2023-11-24T11:49:12.1142950Z io.questdb.cairo.CairoException: [-1] cancelling statement due to user request [fd=7136]
2023-11-24T11:49:12.1143352Z 	at io.questdb.cairo.CairoException.instance(CairoException.java:269)
2023-11-24T11:49:12.1143751Z 	at io.questdb.cairo.CairoException.nonCritical(CairoException.java:114)
2023-11-24T11:49:12.1144153Z 	at io.questdb.cairo.CairoException.queryCancelled(CairoException.java:118)
2023-11-24T11:49:12.1144600Z 	at io.questdb.cairo.sql.NetworkSqlExecutionCircuitBreaker.testTimeout(NetworkSqlExecutionCircuitBreaker.java:189)
2023-11-24T11:49:12.1145114Z 	at io.questdb.cairo.sql.NetworkSqlExecutionCircuitBreaker.statefulThrowExceptionIfTrippedNoThrottle(NetworkSqlExecutionCircuitBreaker.java:175)
2023-11-24T11:49:12.1145647Z 	at io.questdb.griffin.engine.table.AsyncFilteredRecordCursor.calculateSize(AsyncFilteredRecordCursor.java:73)
2023-11-24T11:49:12.1146129Z 	at io.questdb.griffin.engine.table.SelectedRecordCursor.calculateSize(SelectedRecordCursor.java:52)
2023-11-24T11:49:12.1146612Z 	at io.questdb.griffin.engine.LimitRecordCursorFactory$LimitRecordCursor.countRows(LimitRecordCursorFactory.java:308)
2023-11-24T11:49:12.1147107Z 	at io.questdb.griffin.engine.LimitRecordCursorFactory$LimitRecordCursor.countLimit(LimitRecordCursorFactory.java:269)
2023-11-24T11:49:12.1147846Z 	at io.questdb.griffin.engine.LimitRecordCursorFactory$LimitRecordCursor.hasNext(LimitRecordCursorFactory.java:163)
2023-11-24T11:49:12.1148339Z 	at io.questdb.griffin.engine.join.CrossJoinRecordCursorFactory$CrossJoinRecordCursor.hasNext(CrossJoinRecordCursorFactory.java:172)
2023-11-24T11:49:12.1148828Z 	at io.questdb.griffin.engine.table.FilteredRecordCursor.hasNext(FilteredRecordCursor.java:66)
2023-11-24T11:49:12.1149434Z 	at io.questdb.griffin.engine.table.SelectedRecordCursor.hasNext(SelectedRecordCursor.java:80)
2023-11-24T11:49:12.1149870Z 	at io.questdb.griffin.UpdateOperatorImpl.executeUpdate(UpdateOperatorImpl.java:137)
2023-11-24T11:49:12.1150297Z 	at io.questdb.griffin.engine.ops.UpdateOperation.apply(UpdateOperation.java:74)
2023-11-24T11:49:12.1150687Z 	at io.questdb.cairo.TableWriter.apply(TableWriter.java:719)
2023-11-24T11:49:12.1151058Z 	at io.questdb.griffin.CompiledQueryImpl$1.apply(CompiledQueryImpl.java:64)
2023-11-24T11:49:12.1151615Z 	at io.questdb.griffin.CompiledQueryImpl$1.apply(CompiledQueryImpl.java:61)
2023-11-24T11:49:12.1152808Z 	at io.questdb.griffin.engine.ops.OperationDispatcher.execute(OperationDispatcher.java:64)
2023-11-24T11:49:12.1153927Z 	at io.questdb.griffin.CompiledQueryImpl.execute(CompiledQueryImpl.java:100)
2023-11-24T11:49:12.1156002Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.executeUpdate0(PGConnectionContext.java:1608)
2023-11-24T11:49:12.1186615Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.executeUpdate(PGConnectionContext.java:1553)
2023-11-24T11:49:12.1203479Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.processExecute(PGConnectionContext.java:2409)
2023-11-24T11:49:12.1204853Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.processExec(PGConnectionContext.java:2395)
2023-11-24T11:49:12.1206099Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.parse(PGConnectionContext.java:1886)
2023-11-24T11:49:12.1207346Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.handleClientOperation(PGConnectionContext.java:526)
2023-11-24T11:49:12.1208249Z 	at io.questdb.cutlass.pgwire.PGWireServer$1.lambda$$0(PGWireServer.java:79)
2023-11-24T11:49:12.1209271Z 	at io.questdb.network.AbstractIODispatcher.processIOQueue(AbstractIODispatcher.java:199)
2023-11-24T11:49:12.1210338Z 	at io.questdb.cutlass.pgwire.PGWireServer$1.run(PGWireServer.java:109)
2023-11-24T11:49:12.1211348Z 	at io.questdb.mp.Worker.run(Worker.java:148)
2023-11-24T11:49:12.1212644Z 2023-11-24T11:49:12.112625Z I i.q.c.p.WriterPool << [table=`dest~`, thread=7269]
2023-11-24T11:49:12.1213825Z 2023-11-24T11:49:12.112823Z I pg-server connected [ip=127.0.0.1, fd=17112]
2023-11-24T11:49:12.1215280Z 2023-11-24T11:49:12.112886Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.1216501Z 2023-11-24T11:49:12.112895Z I pg-server scheduling disconnect [fd=17112, reason=15]
2023-11-24T11:49:12.1217866Z 2023-11-24T11:49:12.112898Z I pg-server disconnected [ip=127.0.0.1, fd=17112, src=queue]
2023-11-24T11:49:12.1219370Z 2023-11-24T11:49:12.115544Z I pg-server connected [ip=127.0.0.1, fd=16808]
2023-11-24T11:49:12.1220619Z 2023-11-24T11:49:12.115585Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.1226321Z 2023-11-24T11:49:12.115597Z I pg-server scheduling disconnect [fd=16808, reason=15]
2023-11-24T11:49:12.1227618Z 2023-11-24T11:49:12.115600Z I pg-server disconnected [ip=127.0.0.1, fd=16808, src=queue]
2023-11-24T11:49:12.1229194Z 2023-11-24T11:49:12.116912Z I i.q.t.c.p.PGJobContextTest cleaned worker [name=testing, worker=1]
2023-11-24T11:49:12.1230517Z 2023-11-24T11:49:12.116914Z I i.q.t.c.p.PGJobContextTest os scheduled worker stopped [name=testing_1]
2023-11-24T11:49:12.1237268Z 2023-11-24T11:49:12.117134Z I i.q.t.c.p.PGJobContextTest cleaned worker [name=testing, worker=0]
2023-11-24T11:49:12.1239100Z 2023-11-24T11:49:12.117224Z I i.q.t.c.p.PGJobContextTest os scheduled worker stopped [name=testing_0]
2023-11-24T11:49:12.1246839Z 2023-11-24T11:49:12.117398Z I i.q.c.p.WriterPool << [table=`sys.text_import_log~`, thread=7267]
2023-11-24T11:49:12.1248715Z 2023-11-24T11:49:12.117412Z I pg-server disconnected [ip=127.0.0.1, fd=7136, src=shutdown]
2023-11-24T11:49:12.1249699Z 2023-11-24T11:49:12.118372Z I pg-server closed
2023-11-24T11:49:12.1250713Z 2023-11-24T11:49:12.122085Z I i.q.c.TableWriter closed 'sys.text_import_log'
2023-11-24T11:49:12.1252773Z 2023-11-24T11:49:12.122089Z I i.q.c.p.WriterPool closed [table=`sys.text_import_log~`, reason=IDLE, by=7267]
2023-11-24T11:49:12.1264499Z 2023-11-24T11:49:12.123043Z I i.q.c.TableWriter closed 'tab'
2023-11-24T11:49:12.1265863Z 2023-11-24T11:49:12.123046Z I i.q.c.p.WriterPool closed [table=`tab~2`, reason=IDLE, by=7267]
2023-11-24T11:49:12.1267058Z 2023-11-24T11:49:12.123887Z I i.q.c.TableWriter closed 'dest'
2023-11-24T11:49:12.1268264Z 2023-11-24T11:49:12.123890Z I i.q.c.p.WriterPool closed [table=`dest~`, reason=IDLE, by=7267]
2023-11-24T11:49:12.1269462Z 2023-11-24T11:49:12.124152Z I i.q.c.p.ReaderPool closed 'dest~' [at=0:0, reason=IDLE]
2023-11-24T11:49:12.1270769Z 2023-11-24T11:49:12.125245Z I i.q.c.p.ReaderPool closed 'tab~2' [at=0:0, reason=IDLE]
2023-11-24T11:49:12.1271943Z 2023-11-24T11:49:12.125488Z I i.q.c.p.ReaderPool closed 'tab~2' [at=0:1, reason=IDLE]
2023-11-24T11:49:12.1277119Z 2023-11-24T11:49:12.125680Z I i.q.c.w.s.TableSequencerAPI releasing idle table sequencer [tableDir=tab~2]
2023-11-24T11:49:12.1278261Z 2023-11-24T11:49:12.125752Z I i.q.c.p.MetadataPool closed 'dest~' [at=0:0, reason=IDLE]
2023-11-24T11:49:12.1279644Z 2023-11-24T11:49:12.125843Z I i.q.c.p.MetadataPool closed 'dest~' [at=0:1, reason=IDLE]
2023-11-24T11:49:12.1280992Z 2023-11-24T11:49:12.127209Z I i.q.c.w.WalWriter closed 'tab'
2023-11-24T11:49:12.1285886Z 2023-11-24T11:49:12.127211Z I i.q.c.p.WalWriterPool closed 'tab~2' [at=0:0, reason=IDLE]
2023-11-24T11:49:12.1287575Z 2023-11-24T11:49:12.128132Z I i.q.t.AbstractCairoTest Tearing down test PGJobContextTest#testCancelRunningQuery[WITH_WAL]
2023-11-24T11:49:12.1288847Z 2023-11-24T11:49:12.128290Z I i.q.t.AbstractTest Finished test PGJobContextTest#testCancelRunningQuery[WITH_WAL]
2023-11-24T11:49:12.1470768Z 2023-11-24T11:49:12.146244Z I i.q.t.c.p.PGJobContextTest asserting PG Wire server [mode=EXTENDED, binary=true, prepareThreshold=-1]
2023-11-24T11:49:12.1472681Z 2023-11-24T11:49:12.146255Z I i.q.t.AbstractTest Starting test PGJobContextTest#testCancelRunningQuery[WITH_WAL]
2023-11-24T11:49:12.1477901Z 2023-11-24T11:49:12.147458Z I i.q.t.s.TestFilesFacadeImpl cannot remove file: C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\tables.d.0, errno:2
2023-11-24T11:49:12.1494719Z 2023-11-24T11:49:12.149037Z I i.q.c.TableNameRegistryStore reloading tables file [path=C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\tables.d.0]
2023-11-24T11:49:12.1500091Z 2023-11-24T11:49:12.149631Z I i.q.g.SqlCompilerImpl parse [fd=-1, q=CREATE TABLE IF NOT EXISTS "sys.text_import_log" (ts timestamp, id string, table_name symbol, file symbol, phase symbol, status symbol, message string,rows_handled long,rows_imported long,errors long) timestamp(ts) partition by DAY BYPASS WAL]
2023-11-24T11:49:12.1507319Z 2023-11-24T11:49:12.150441Z I i.q.c.CairoEngine locked [table=`sys.text_import_log~`, thread=7267]
2023-11-24T11:49:12.1982341Z 2023-11-24T11:49:12.197342Z I i.q.c.p.WriterPool created [table=`sys.text_import_log~`, thread=7267]
2023-11-24T11:49:12.1984062Z 2023-11-24T11:49:12.197374Z I i.q.c.TableWriter open 'sys.text_import_log'
2023-11-24T11:49:12.2104828Z 2023-11-24T11:49:12.208428Z I i.q.c.CairoEngine unlocked [table=`sys.text_import_log`]
2023-11-24T11:49:12.2105540Z 2023-11-24T11:49:12.208461Z I i.q.c.p.WriterPool >> [table=`sys.text_import_log~`, thread=7267]
2023-11-24T11:49:12.2106854Z 2023-11-24T11:49:12.208759Z A pg-server listening on 0.0.0.0:0 [fd=10252 backlog=64]
2023-11-24T11:49:12.2120277Z 2023-11-24T11:49:12.209097Z I i.q.t.c.p.PGJobContextTest worker pool started [pool=testing]
2023-11-24T11:49:12.2123985Z 2023-11-24T11:49:12.210683Z I i.q.t.c.p.PGJobContextTest os scheduled worker started [name=testing_0]
2023-11-24T11:49:12.2130242Z 2023-11-24T11:49:12.211081Z I pg-server connected [ip=127.0.0.1, fd=1576]
2023-11-24T11:49:12.2131987Z 2023-11-24T11:49:12.211516Z I i.q.t.c.p.PGJobContextTest os scheduled worker started [name=testing_1]
2023-11-24T11:49:12.2133438Z 2023-11-24T11:49:12.211783Z I i.q.g.SqlCompilerImpl parse [fd=1576, q=SET extra_float_digits = 3]
2023-11-24T11:49:12.2134859Z 2023-11-24T11:49:12.212357Z I i.q.g.SqlCompilerImpl parse [fd=1576, q=SET application_name = 'PostgreSQL JDBC Driver']
2023-11-24T11:49:12.2141131Z 2023-11-24T11:49:12.212643Z I i.q.g.SqlCompilerImpl parse [fd=-1, q=create table if not exists tab as (select x::timestamp ts, x, rnd_double() d from long_sequence(1000000)) timestamp(ts) partition by day]
2023-11-24T11:49:12.2143270Z 2023-11-24T11:49:12.213623Z I i.q.c.CairoEngine locked [table=`tab~`, thread=7267]
2023-11-24T11:49:12.2331894Z 2023-11-24T11:49:12.232472Z I i.q.c.TableWriter open 'tab'
2023-11-24T11:49:12.2420047Z 2023-11-24T11:49:12.241579Z I i.q.c.TableWriter switched partition [path='C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\tab~\1970-01-01']
2023-11-24T11:49:12.3665437Z 2023-11-24T11:49:12.357308Z I i.q.c.CairoEngine unlocked [table=`tab`]
2023-11-24T11:49:12.3667100Z 2023-11-24T11:49:12.357378Z I i.q.g.SqlCompilerImpl parse [fd=-1, q=create table if not exists dest as (select x l from long_sequence(10000))]
2023-11-24T11:49:12.3668495Z 2023-11-24T11:49:12.358448Z I i.q.c.CairoEngine locked [table=`dest~`, thread=7267]
2023-11-24T11:49:12.3830871Z 2023-11-24T11:49:12.375408Z I i.q.c.TableWriter open 'dest'
2023-11-24T11:49:12.3833304Z 2023-11-24T11:49:12.378199Z I i.q.c.TableWriter switched partition [path='C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\dest~\default']
2023-11-24T11:49:12.3835037Z 2023-11-24T11:49:12.382257Z I i.q.c.CairoEngine unlocked [table=`dest`]
2023-11-24T11:49:12.4238256Z 2023-11-24T11:49:12.401626Z I pg-server connected [ip=127.0.0.1, fd=16204]
2023-11-24T11:49:12.4249404Z 2023-11-24T11:49:12.401749Z I i.q.g.SqlCompilerImpl parse [fd=1576, q=create table new_tab as (select count(*) from tab t1 join tab t2 on t1.x = t2.x where t1.x > 0)]
2023-11-24T11:49:12.4250947Z 2023-11-24T11:49:12.403705Z I i.q.g.SqlCodeGenerator JIT enabled for (sub)query [tableName=t1, fd=1576]
2023-11-24T11:49:12.4252790Z 2023-11-24T11:49:12.405308Z I i.q.c.CairoEngine locked [table=`new_tab~`, thread=7275]
2023-11-24T11:49:12.4254094Z 2023-11-24T11:49:12.417111Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.4255301Z 2023-11-24T11:49:12.417132Z I pg-server scheduling disconnect [fd=16204, reason=15]
2023-11-24T11:49:12.4265147Z 2023-11-24T11:49:12.417140Z I pg-server disconnected [ip=127.0.0.1, fd=16204, src=queue]
2023-11-24T11:49:12.4276380Z 2023-11-24T11:49:12.419562Z I i.q.c.TableWriter open 'new_tab'
2023-11-24T11:49:12.4277834Z 2023-11-24T11:49:12.422251Z I i.q.c.TableWriter switched partition [path='C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\new_tab~\default']
2023-11-24T11:49:12.4279392Z 2023-11-24T11:49:12.423412Z I pg-server connected [ip=127.0.0.1, fd=15864]
2023-11-24T11:49:12.4280952Z 2023-11-24T11:49:12.423477Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.4282336Z 2023-11-24T11:49:12.423490Z I pg-server scheduling disconnect [fd=15864, reason=15]
2023-11-24T11:49:12.4283534Z 2023-11-24T11:49:12.423495Z I pg-server disconnected [ip=127.0.0.1, fd=15864, src=queue]
2023-11-24T11:49:12.4292779Z 2023-11-24T11:49:12.426237Z I pg-server connected [ip=127.0.0.1, fd=15864]
2023-11-24T11:49:12.6393497Z 2023-11-24T11:49:12.426278Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:12.8436506Z 2023-11-24T11:49:12.426289Z I pg-server scheduling disconnect [fd=15864, reason=15]
2023-11-24T11:49:13.0456183Z 2023-11-24T11:49:12.426292Z I pg-server disconnected [ip=127.0.0.1, fd=15864, src=queue]
2023-11-24T11:49:13.2405182Z 2023-11-24T11:49:12.428932Z I pg-server connected [ip=127.0.0.1, fd=15864]
2023-11-24T11:49:13.4442117Z 2023-11-24T11:49:12.428972Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:13.4770144Z 2023-11-24T11:49:12.428984Z I pg-server scheduling disconnect [fd=15864, reason=15]
2023-11-24T11:49:13.6458010Z 2023-11-24T11:49:12.428987Z I pg-server disconnected [ip=127.0.0.1, fd=15864, src=queue]
2023-11-24T11:49:13.6463995Z 2023-11-24T11:49:12.431299Z I i.q.c.TableReader open partition C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\tab~\1970-01-01 [rowCount=1000000, partitionNameTxn=-1, transientRowCount=1000000, partitionIndex=0, partitionCount=1]
2023-11-24T11:49:13.6465349Z 2023-11-24T11:49:12.431632Z E i.q.g.SqlCompilerImpl could not create table [error=
2023-11-24T11:49:13.6570751Z io.questdb.cairo.CairoException: [-1] cancelling statement due to user request [fd=1576]
2023-11-24T11:49:13.6572344Z 	at io.questdb.cairo.CairoException.instance(CairoException.java:269)
2023-11-24T11:49:13.6573122Z 	at io.questdb.cairo.CairoException.nonCritical(CairoException.java:114)
2023-11-24T11:49:13.6573850Z 	at io.questdb.cairo.CairoException.queryCancelled(CairoException.java:118)
2023-11-24T11:49:13.6574613Z 	at io.questdb.cairo.sql.NetworkSqlExecutionCircuitBreaker.testTimeout(NetworkSqlExecutionCircuitBreaker.java:189)
2023-11-24T11:49:13.6575482Z 	at io.questdb.cairo.sql.NetworkSqlExecutionCircuitBreaker.statefulThrowExceptionIfTrippedNoThrottle(NetworkSqlExecutionCircuitBreaker.java:175)
2023-11-24T11:49:13.6578118Z 	at io.questdb.cairo.sql.NetworkSqlExecutionCircuitBreaker.statefulThrowExceptionIfTripped(NetworkSqlExecutionCircuitBreaker.java:168)
2023-11-24T11:49:13.6578941Z 	at io.questdb.cairo.TableUtils.populateRowIDHashMap(TableUtils.java:1214)
2023-11-24T11:49:13.6579806Z 	at io.questdb.griffin.engine.join.HashJoinLightRecordCursorFactory$HashJoinRecordCursor.buildMapOfSlaveRecords(HashJoinLightRecordCursorFactory.java:193)
2023-11-24T11:49:13.6580684Z 	at io.questdb.griffin.engine.join.HashJoinLightRecordCursorFactory$HashJoinRecordCursor.size(HashJoinLightRecordCursorFactory.java:176)
2023-11-24T11:49:13.6581538Z 	at io.questdb.griffin.engine.groupby.CountRecordCursorFactory$CountRecordCursor.hasNext(CountRecordCursorFactory.java:109)
2023-11-24T11:49:13.6583475Z 	at io.questdb.griffin.SqlCompilerImpl.copyUnordered(SqlCompilerImpl.java:1763)
2023-11-24T11:49:13.6584186Z 	at io.questdb.griffin.SqlCompilerImpl.copyTableData(SqlCompilerImpl.java:1647)
2023-11-24T11:49:13.6584885Z 	at io.questdb.griffin.SqlCompilerImpl.copyTableDataAndUnlock(SqlCompilerImpl.java:1688)
2023-11-24T11:49:13.6585769Z 	at io.questdb.griffin.SqlCompilerImpl.createTableFromCursorExecutor(SqlCompilerImpl.java:1892)
2023-11-24T11:49:13.6586701Z 	at io.questdb.griffin.SqlCompilerImpl.createTable(SqlCompilerImpl.java:1842)
2023-11-24T11:49:13.6587363Z 	at io.questdb.griffin.SqlCompilerImpl.executeWithRetries(SqlCompilerImpl.java:1943)
2023-11-24T11:49:13.6588058Z 	at io.questdb.griffin.SqlCompilerImpl.createTableWithRetries(SqlCompilerImpl.java:1927)
2023-11-24T11:49:13.6589147Z 	at io.questdb.griffin.SqlCompilerImpl.compileUsingModel(SqlCompilerImpl.java:1441)
2023-11-24T11:49:13.6590021Z 	at io.questdb.griffin.SqlCompilerImpl.compileInner(SqlCompilerImpl.java:1405)
2023-11-24T11:49:13.6590729Z 	at io.questdb.griffin.SqlCompilerImpl.compile(SqlCompilerImpl.java:229)
2023-11-24T11:49:13.6591420Z 	at io.questdb.cairo.pool.SqlCompilerPool$C.compile(SqlCompilerPool.java:98)
2023-11-24T11:49:13.6592288Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.compileQuery(PGConnectionContext.java:1332)
2023-11-24T11:49:13.6592959Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.parseQueryText(PGConnectionContext.java:1932)
2023-11-24T11:49:13.6593697Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.processParse(PGConnectionContext.java:2445)
2023-11-24T11:49:13.6594391Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.parse(PGConnectionContext.java:1867)
2023-11-24T11:49:13.6595080Z 	at io.questdb.cutlass.pgwire.PGConnectionContext.handleClientOperation(PGConnectionContext.java:526)
2023-11-24T11:49:13.6595981Z 	at io.questdb.cutlass.pgwire.PGWireServer$1.lambda$$0(PGWireServer.java:79)
2023-11-24T11:49:13.6596668Z 	at io.questdb.network.AbstractIODispatcher.processIOQueue(AbstractIODispatcher.java:199)
2023-11-24T11:49:13.6597769Z 	at io.questdb.cutlass.pgwire.PGWireServer$1.run(PGWireServer.java:109)
2023-11-24T11:49:13.6598430Z 	at io.questdb.mp.Worker.run(Worker.java:148)
2023-11-24T11:49:13.6599016Z ]
2023-11-24T11:49:13.6599966Z 2023-11-24T11:49:12.432356Z I i.q.c.TableWriter closed 'new_tab'
2023-11-24T11:49:13.6600849Z 2023-11-24T11:49:12.432576Z I i.q.c.CairoEngine unlocked [table=`new_tab`]
2023-11-24T11:49:13.6602155Z 2023-11-24T11:49:12.432588Z E i.q.g.SqlCompilerImpl cancelling statement due to user request [fd=1576] [errno=-1]
2023-11-24T11:49:13.6603155Z 2023-11-24T11:49:12.432907Z I i.q.c.CairoEngine locked [table=`new_tab~`, thread=7275]
2023-11-24T11:49:13.6603996Z 2023-11-24T11:49:12.434797Z I pg-server connected [ip=127.0.0.1, fd=4236]
2023-11-24T11:49:13.6605086Z 2023-11-24T11:49:12.434859Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:13.6671138Z 2023-11-24T11:49:12.434867Z I pg-server scheduling disconnect [fd=4236, reason=15]
2023-11-24T11:49:13.6673463Z 2023-11-24T11:49:12.434871Z I pg-server disconnected [ip=127.0.0.1, fd=4236, src=queue]
2023-11-24T11:49:13.6757038Z 2023-11-24T11:49:12.443759Z I pg-server connected [ip=127.0.0.1, fd=4236]
2023-11-24T11:49:13.6771266Z 2023-11-24T11:49:12.446981Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:13.6773169Z 2023-11-24T11:49:12.447003Z I pg-server scheduling disconnect [fd=4236, reason=15]
2023-11-24T11:49:13.6831712Z 2023-11-24T11:49:12.447013Z I pg-server disconnected [ip=127.0.0.1, fd=4236, src=queue]
2023-11-24T11:49:13.6841211Z 2023-11-24T11:49:12.448001Z I i.q.g.SqlCompilerImpl parse [fd=1576, q=select count(*) from tab t1 join tab t2 on t1.x = t2.x where t1.x > 0]
2023-11-24T11:49:13.6843200Z 2023-11-24T11:49:12.448807Z I i.q.g.SqlCompilerImpl plan [q=`select-group-by count() count from (select [x] from tab t1 timestamp (ts) join select [x] from tab t2 timestamp (ts) on t2.x = t1.x where x > 0) t1`, fd=1576]
2023-11-24T11:49:13.6907150Z 2023-11-24T11:49:12.449141Z I i.q.g.SqlCodeGenerator JIT enabled for (sub)query [tableName=t1, fd=1576]
2023-11-24T11:49:13.6911366Z 2023-11-24T11:49:12.449769Z I i.q.c.p.PGConnectionContext prepare [name=S_2]
2023-11-24T11:49:13.6913035Z 2023-11-24T11:49:12.450319Z I i.q.c.p.PGConnectionContext query cache used [fd=1576]
2023-11-24T11:49:13.6914786Z 2023-11-24T11:49:13.304472Z I i.q.c.TableReader open partition C:\Users\VSSADM~1\AppData\Local\Temp\junit16367896118120204858\dbRoot\tab~\1970-01-01 [rowCount=1000000, partitionNameTxn=-1, transientRowCount=1000000, partitionIndex=0, partitionCount=1]
2023-11-24T11:49:13.6917425Z 2023-11-24T11:49:13.444558Z I pg-server connected [ip=127.0.0.1, fd=8392]
2023-11-24T11:49:13.6919178Z 2023-11-24T11:49:13.444697Z I i.q.c.p.CleartextPasswordPgWireAuthenticator cancel request [pid=63]
2023-11-24T11:49:13.6920958Z 2023-11-24T11:49:13.444779Z I pg-server scheduling disconnect [fd=8392, reason=15]
2023-11-24T11:49:13.6922457Z 2023-11-24T11:49:13.444787Z I pg-server disconnected [ip=127.0.0.1, fd=8392, src=queue]
2023-11-24T11:49:13.6924065Z 2023-11-24T11:49:13.618736Z I pg-server scheduling disconnect [fd=1576, reason=15]
2023-11-24T11:49:13.6931857Z 2023-11-24T11:49:13.618809Z I i.q.t.c.p.PGJobContextTest cleaned worker [name=testing, worker=0]
2023-11-24T11:49:13.6934601Z 2023-11-24T11:49:13.618812Z I i.q.t.c.p.PGJobContextTest os scheduled worker stopped [name=testing_0]
2023-11-24T11:49:13.6936552Z 2023-11-24T11:49:13.619101Z I i.q.t.c.p.PGJobContextTest cleaned worker [name=testing, worker=1]
2023-11-24T11:49:13.6938222Z 2023-11-24T11:49:13.619216Z I i.q.c.p.WriterPool << [table=`sys.text_import_log~`, thread=7267]
2023-11-24T11:49:13.6939661Z 2023-11-24T11:49:13.619246Z I pg-server disconnected [ip=127.0.0.1, fd=1576, src=queue]
2023-11-24T11:49:13.6941035Z 2023-11-24T11:49:13.619487Z I pg-server closed
2023-11-24T11:49:13.6942285Z 2023-11-24T11:49:13.620223Z I i.q.c.p.ReaderPool closed 'tab~' [at=0:0, reason=POOL_CLOSED]
2023-11-24T11:49:13.6943938Z 2023-11-24T11:49:13.621059Z I i.q.c.p.ReaderPool closed 'tab~' [at=0:1, reason=POOL_CLOSED]
2023-11-24T11:49:13.6945548Z 2023-11-24T11:49:13.624257Z I i.q.c.TableWriter closed 'sys.text_import_log'
2023-11-24T11:49:13.6946995Z 2023-11-24T11:49:13.624261Z I i.q.c.p.WriterPool closed [table=`sys.text_import_log~`, reason=POOL_CLOSED, by=7267]
2023-11-24T11:49:13.6948701Z 2023-11-24T11:49:13.625205Z I i.q.c.TableWriter closed 'dest'
2023-11-24T11:49:13.6950205Z 2023-11-24T11:49:13.625208Z I i.q.c.p.WriterPool closed [table=`dest~`, reason=POOL_CLOSED, by=7267]
2023-11-24T11:49:13.6952880Z 2023-11-24T11:49:13.626701Z I i.q.c.TableWriter closed 'tab'
2023-11-24T11:49:13.6956516Z 2023-11-24T11:49:13.626704Z I i.q.c.p.WriterPool closed [table=`tab~`, reason=POOL_CLOSED, by=7267]
2023-11-24T11:49:13.6957638Z 2023-11-24T11:49:13.626739Z I i.q.t.AbstractCairoTest Tearing down test PGJobContextTest#testCancelRunningQuery[WITH_WAL]
2023-11-24T11:49:13.6960049Z 2023-11-24T11:49:13.626904Z I i.q.t.AbstractTest Finished test PGJobContextTest#testCancelRunningQuery[WITH_WAL]
2023-11-24T11:49:13.6961866Z 2023-11-24T11:49:13.637673Z I i.q.t.AbstractCairoTest Tearing down test PGJobContextTest#testCancelRunningQuery[WITH_WAL]
2023-11-24T11:49:13.6964486Z 2023-11-24T11:49:13.637693Z I i.q.t.AbstractTest Finished test PGJobContextTest#testCancelRunningQuery[WITH_WAL]
2023-11-24T11:49:13.6965770Z 2023-11-24T11:49:13.638221Z I i.q.t.c.p.PGJobContextTest os scheduled worker stopped [name=testing_1]
2023-11-24T11:49:13.6972724Z Exception in thread "cancellation thread" java.lang.RuntimeException: org.postgresql.util.PSQLException: This connection has been closed.
2023-11-24T11:49:13.6975141Z 	at io.questdb.test.cutlass.pgwire.PGJobContextTest.lambda$testCancelRunningQuery$15(PGJobContextTest.java:2375)
2023-11-24T11:49:13.6976109Z 	at java.base/java.lang.Thread.run(Thread.java:829)
2023-11-24T11:49:13.6977266Z Caused by: org.postgresql.util.PSQLException: This connection has been closed.
2023-11-24T11:49:13.6980353Z 	at org.postgresql.jdbc.PgConnection.checkClosed(PgConnection.java:907)
2023-11-24T11:49:13.6981599Z 	at org.postgresql.jdbc.PgConnection.cancelQuery(PgConnection.java:1119)
2023-11-24T11:49:13.6982895Z 	at io.questdb.test.cutlass.pgwire.PGJobContextTest.lambda$testCancelRunningQuery$15(PGJobContextTest.java:2372)
2023-11-24T11:49:13.6984129Z 	... 1 more
2023-11-24T11:49:13.6985004Z random seeds: 1483341932000L, 1700826553645L
2023-11-24T11:49:13.6986238Z 2023-11-24T11:49:13.649721Z E i.q.t.TestListener ***** Test Failed *****io.questdb.test.cutlass.pgwire.PGJobContextTest.testCancelRunningQuery[WITH_WAL] duration_ms=1937 : 
2023-11-24T11:49:13.6988193Z java.lang.AssertionError: expected PSQLException with cancel message for query: select count(*) from tab t1 join tab t2 on t1.x = t2.x where t1.x > 0
2023-11-24T11:49:13.6989266Z 	at org.junit.Assert.fail(Assert.java:89)
2023-11-24T11:49:13.6990263Z 	at io.questdb.test.cutlass.pgwire.PGJobContextTest.lambda$testCancelRunningQuery$16(PGJobContextTest.java:2382)
2023-11-24T11:49:13.6991371Z 	at io.questdb.test.cutlass.pgwire.PGJobContextTest.lambda$assertWithPgServer$169(PGJobContextTest.java:9800)
2023-11-24T11:49:13.6994060Z 	at io.questdb.test.AbstractCairoTest.lambda$assertMemoryLeak$7(AbstractCairoTest.java:949)
2023-11-24T11:49:13.6995274Z 	at io.questdb.test.tools.TestUtils.assertMemoryLeak(TestUtils.java:610)
2023-11-24T11:49:13.6996801Z 	at io.questdb.test.AbstractCairoTest.assertMemoryLeak(AbstractCairoTest.java:946)
2023-11-24T11:49:13.6998096Z 	at io.questdb.test.AbstractCairoTest.assertMemoryLeak(AbstractCairoTest.java:940)
2023-11-24T11:49:13.6999786Z 	at io.questdb.test.cutlass.pgwire.PGJobContextTest.assertWithPgServer(PGJobContextTest.java:9793)
2023-11-24T11:49:13.7000899Z 	at io.questdb.test.cutlass.pgwire.PGJobContextTest.assertWithPgServer(PGJobContextTest.java:9757)
2023-11-24T11:49:13.7002263Z 	at io.questdb.test.cutlass.pgwire.PGJobContextTest.assertWithPgServer(PGJobContextTest.java:9741)
2023-11-24T11:49:13.7003356Z 	at io.questdb.test.cutlass.pgwire.PGJobContextTest.testCancelRunningQuery(PGJobContextTest.java:2356)
2023-11-24T11:49:13.7004481Z 	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2023-11-24T11:49:13.7005717Z 	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2023-11-24T11:49:13.7006871Z 	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2023-11-24T11:49:13.7007817Z 	at java.lang.reflect.Method.invoke(Method.java:566)
2023-11-24T11:49:13.7010587Z 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
2023-11-24T11:49:13.7011876Z 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
2023-11-24T11:49:13.7013086Z 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
2023-11-24T11:49:13.7016139Z 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
2023-11-24T11:49:13.7018449Z 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
2023-11-24T11:49:13.7020892Z 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
2023-11-24T11:49:13.7022533Z 	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
2023-11-24T11:49:13.7023877Z 	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:299)
2023-11-24T11:49:13.7025083Z 	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:293)
2023-11-24T11:49:13.7026246Z 	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
2023-11-24T11:49:13.7027176Z 	at java.lang.Thread.run(Thread.java:829)
2023-11-24T11:49:13.7033899Z 2023-11-24T11:49:13.649809Z I i.q.t.TestListener <<<< io.questdb.test.cutlass.pgwire.PGJobContextTest.testCancelRunningQuery[WITH_WAL] duration_ms=1937
```

--- a/core/src/test/java/io/questdb/test/cutlass/pgwire/PGJobContextTest.java
+++ b/core/src/test/java/io/questdb/test/cutlass/pgwire/PGJobContextTest.java
@@ -2355,7 +2355,7 @@ if __name__ == "__main__":
 
         assertWithPgServer(CONN_AWARE_EXTENDED_BINARY, (connection, binary) -> {
             ddl("create table if not exists tab as " +
-                    "(select x::timestamp ts, x, rnd_double() d from long_sequence(1000000)) timestamp(ts) partition by day");
+                    "(select x::timestamp ts, x, rnd_double() d from long_sequence(5000000)) timestamp(ts) partition by day");
             ddl("create table if not exists dest as (select x l from long_sequence(10000))");
             mayDrainWalQueue();
 

